hw/dv/dpi/usbdpi/usbdpi.c

```diff
@@ -756,8 +756,8 @@ char usbdpi_host_to_device(void *ctx_void, const svBitVecVal *usb_d2p) {
   int dat;
 
   if (ctx->tick == 0) {
-    int i;
 #ifdef NEED_SLEEP
+    int i;
```

Comment:
Why not define this inside the for loop?

Reply:
I was just fixing an 'unused variable' warning by moving the declaration into the #ifdef-controlled section where it's used. The 'NEED_SLEEP' case is apparently legacy, but best kept around until the DPI model no longer has fixed timing